app/assets/javascripts/components/select_experience_domain.js

```diff
@@ -18,6 +24,16 @@ type State = {
 };
 
 class SelectExperienceDomain extends React.PureComponent<Props, State> {
+  static defaultProps = {
+    mode: "default",
+    alreadyUsedDomains: [],
+    onChange: () => {},
+    onSelect: () => {},
+    value: null,
+    notFoundContent: null,
+    disabled: false,
```

Comment:
Sorry to bug you again about that, but I think that `alreadyUsedDomains` is the only prop which needs to get a default here. All other props should be defined as nullable in the props type (e.g., `mode?: string`) which is why they don't need a default value. Does that work?